src/shared/tpm2-util.h

```diff
@@ -70,6 +70,8 @@ typedef struct {
 
         TPML_ALG_PROPERTY capability_algs;
         TPML_PCR_SELECTION capability_pcrs;
+
+        TPMT_SYM_DEF session_symmetric_template;
```

Comment:
hmm, why is this a variable field? 

i mean, if we only support one combination for this not sure why this should be a vraiable field here instead of static data as before.

or do you intend to add support for stronger encryption for this sooner or later? what's the rationale here?

Reply:
Initial thought was to allow multiple options for this, as you say possibly for stronger encryption. However for now it could certainly just verify static data instead of storing it, I'll update it to do that (and changing to a variable later, if needed for multiple options, should be trivial)